Code/Framework/AzToolsFramework/Tests/aztoolsframeworktests_files.cmake

```diff
@@ -74,7 +74,7 @@ set(FILES
     Prefab/PrefabEntityAliasTests.cpp
     Prefab/PrefabInstanceToTemplatePropagatorTests.cpp
     Prefab/PrefabInstantiateTests.cpp
-    Prefab/PrefabInstantiateTests.cpp
+    Prefab/PrefabAssetFixupTests.cpp
```

Comment:
This doesn't sound correct. This will cause 'PrefabInstantiateTests' to be not generated as part of the solution